packages/web3torrent/src/contexts/budget-context.ts

```diff
@@ -0,0 +1,47 @@
+import {web3torrent} from '../clients/web3torrent-client';
+import {INITIAL_BUDGET_AMOUNT} from '../constants';
+import {useState, useEffect} from 'react';
+
+import React from 'react';
+
+export const BudgetContext = React.createContext<ReturnType<typeof useBudgetContext>>(undefined);
+
+export function useBudgetContext({initializationContext}) {
+  const {paymentChannelClient} = web3torrent;
+
+  const [loading, setFetching] = useState(true);
```

Comment:
I think it might be more conventional
```suggestion
  const [loading, setLoading] = useState(true);
```

Reply:
Yup, just forgot to rename the setter.